certbot/plugins/dns_common_lexicon.py

```diff
@@ -7,6 +7,11 @@
 from certbot import errors
 from certbot.plugins import dns_common
 
+try:
+    from lexicon.config import ConfigResolver
+except ImportError:
+    ConfigResolver = None  # type: ignore
```

Comment:
When would an `ImportError` be raised here? Why do we want to ignore this?

Reply:
An `ImportError` could happen in theory because this code is in the certbot core part, and the core part does not require Lexicon.

We can ignore it because in effect, this code is called by a plugin that requires Lexicon, so the `ImportError` will not occur.

And yes definitively I should add a comment about this.